components/controls/selectors/OrganizationContactSelector.vue

```diff
@@ -82,14 +83,17 @@ export default {
   mounted: load,
   methods: {
     fullName,
+    ...mapActions({
+      getOrganizationContacts: 'organizations/getOrganizationContacts',
+    }),
   },
 };
 
 function fullName({ firstName, lastName, designation }) {
   return _d.remove(`${firstName} ${lastName} ${designation && `(${designation.trim()})`}`);
 }
 async function load() {
-  const contacts      = this.disabled ? [] : await this.$kronosApi.getOrganizationContacts(this.organizationId);
+  const contacts      = this.disabled ? [] : await this.getOrganizationContacts(this.organizationId);
```

Comment:
shoudl pass useCache=true
```suggestion
  const contacts      = this.disabled ? [] : await this.getOrganizationContacts(this.organizationId, true);
```

Reply:
DONE! changes done as suggested. passed parameter in one object form. as store actions take parameter in single object.